packages/node/.gitignore

```diff
@@ -9,5 +9,3 @@ coverage/
 # Config files
 secrets.env
 config.json
-**/secrets.env
```

Comment:
Maybe this could be fixed for all .gitignores in airnode monorepo? (in a separate PR?)